app/controllers/organizations_controller.rb

```diff
@@ -1,20 +1,12 @@
 class OrganizationsController < ApplicationController
-  before_filter :load_resource
-
-  def load_resource
-    if params[:id]
-      @organization = Organization.find(params[:id])
-    else
-      @organizations = Organization.all
-    end
-  end
+  before_filter :load_resource, except: [:new, :index, :create]
```

Comment:
what about using `only` instead? It will feel a bit more under control. Loading it by default in any new action seems dangerous to me.